api/src/opentrons/protocol_engine/state/modules.py

```diff
@@ -609,9 +609,13 @@ def get_requested_model(self, module_id: str) -> Optional[ModuleModel]:
         except KeyError as e:
             raise errors.ModuleNotLoadedError(module_id=module_id) from e
 
+    # TODO(jbl 2023-06-20) rename this method to better reflect it's not just "connected" modules
     def get_connected_model(self, module_id: str) -> ModuleModel:
```

Comment:
maybe we should change this to 
```suggestion
    def get_loaded_model(self, module_id: str) -> ModuleModel:
```

Reply:
not for this pr